global/scripts/TopupPreprocessingAll.sh

```diff
@@ -216,7 +217,7 @@ if [[ $UnwarpDir = "x" || $UnwarpDir = "x-" || $UnwarpDir = "-x" ]] ; then
     i=`echo "$i + 1" | bc`
   done
 # Y direction phase encode
-elif [[ $UnwarpDir = "y" || $UnwarpDir = "y-" || $UnwarpDir = "-y" ]] ; then
+elif [[ $UnwarpDir = [yj] || $UnwarpDir = [yj]- || $UnwarpDir = -[yj] ]] ; then
```

Comment:
What happens in this script if the unwarp direction is specified as z/-z or k/-k or 'q' for that matter? 
I think it will neither enter the unwarpdir = [xi] [xi]- -[xi] block nor the unwarpdir = [yj] [yj]- -[yj] block. Is that the right thing to do, do nothing and then move on? Is this be an error that is captured and reported?

Reply:
Just changed the script to error out in that situation.